packages/harmony/src/components/button/BaseButton/BaseButton.module.css

```diff
@@ -34,4 +34,5 @@
 
 .fullWidth {
   width: 100%;
+  flex-shrink: 1;
```

Comment:
not sure we would want this. why was it necessary?

Reply:
to be able to fit 2 buttons in a row. with `width: 100%` only the first button took up entire width and pushed other button off screen. Had to wrap each in a `div` with `fles-shrink: 1` to get correct behavior but then figured this made more sense? i think it doesn't affect normal single-row button. open to suggestions tho.

Reply:
You should be able to put them next to each other 🤔 
https://jsfiddle.net/o7kj4ufm/

Reply:
Oh i see, on the .button itself, it's set to flex-shrink: 0, probably on purposes. This change lgtm